src/Themes.ts

```diff
@@ -1,12 +1,15 @@
-import {redrawProof, redrawTree} from "./SharedToolUtils/DrawUtils";
-import {TreeContext} from "./TreeContext";
-
 /**
  * Provides utilities for extracting important colors for the application from the CSS themes.
- *
  * @author James Oswald
  */
 
+// Import and initialize the theme-change library.
+import {themeChange} from "theme-change";
+themeChange();
```

Comment:
Not sure what this does or why it is here. I know it is a module that we added but it is placed very strangely and I cannot be sure what it is changing